docs/style.css

```diff
@@ -16,893 +16,76 @@ You should have received a copy of the GNU General Public License
 along with this program. If not, see <https://www.gnu.org/licenses/>.
 */
 
+html, body {
+    margin-left:10%;
+    margin-right:10%;
+    font-family: courier, mono;
+    font-size: 15px;
+}
+
 img {
 	width: 8em;
 	height: auto;
 }
 
-h1.title {
-	margin: auto;
-	padding: 1em;
-}
-
-div.logo {
+.logo {
 	display: flex;
 	align-items: center;
+	margin-bottom: 5px;
 }
 
-#course_links {
-	text-align: center;
-}
-
-#confusion {
-	margin: 0 auto;
-}
-
-#confusion > img {
-	display: block;
-	margin-left: auto;
-	margin-right: auto;
-	width: 90%;
-	height: auto;
+h1.title {
+	margin: auto;
+	padding: 1em;
 }
 
 hr {
 	background: #e83d2a;
+	color: #e83d2a;
 	height: 2px;
 	margin: 0;
 }
 
-body {
-	padding-top: 10px;
-	display: block;
-	margin: auto;
-	width: 45vw;
-	min-width: 60vw;
-	font-family: courier;
-	border-collapse: collapse;
+.nav {
+	display: flex;
+	flex-wrap: wrap;
+	text-align: center;
+	box-sizing: border-box;
 }
 
 a {
 	color: #2db8d4;
 }
-
 a:visited {
 	color: #005d70;
 }
 
-p {
-	font-size: 11pt;
-}
-
-h5 {
-	font-size: 14pt;
-	margin: 0;
-}
-
-table {
-	border-collapse: collapse;
-}
-
-table, th, td {
-	border: 1px solid;
-	padding: 0.5em;
-}
-
-pre {
-	white-space: pre-wrap;
-}
-
-code {
-	background-color: #eeeeee;
-}
-
-input {
-	border: none;
-	border-bottom: 2px solid #e83d2a;
-	border-radius: 0;
-	background-color: #ffffcc;
-	font-size: 1.3em;
-	font-family: courier;
-}
-
-input.login.logout[type="submit"] {
-	width: 4.5em;
-	height: 1.8em;
-	font-size: 1.5em;
-}
-
-input.form {
-	float: right;
-}
-
-
-form.login input {
-	width: 30em;
-	height: 2em;
-}
-
-@counter-style hex {
-	system: alphabetic;
-	symbols: '0' '1' '2' '3' '4' '5' '6' '7' '8' '9' 'A' 'B' 'C' 'D' 'E' 'F';
-}
-
-ol {
-	counter-reset: count 16;
-}
-
-ol > li {
-	counter-increment: count;
-}
-
-ol > li::marker {
-	content: '0x' counter(count, hex) ': ';
-}
-
-blockquote {
-	position: relative;
-}
-
-blockquote::before {
-	position: absolute;
-	content: open-quote;
-	font-size: 4em;
-	margin-left: -0.6em;
-	margin-top: -0.4em;
-}
-
-blockquote::after {
-	position: absolute;
-	content: close-quote;
-	font-size: 4em;
-	bottom: 0;
-	right: 0;
-	margin-right: -0.6em;
-	margin-bottom: -0.8em;
-}
-
-blockquote * {
-	display: inline;
-}
-
-div.nav {
-	display: flex;
-	text-align: center;
-	box-sizing: border-box;
-}
-
 a.nav {
 	padding: 0.5em;
-	font-size: 1em;
+	font-size: 1.3em;
 	border-left: 1px solid black;
 	background-color: #eeeeee;
-}
-
-a.nav:first-child {
-	border-left: 0;
+	color: #005d70;
+	text-decoration: none;
 }
 
 a.nav:last-child {
-	border-right: 0;
-}
-
-label {
-	font-size: 1.5em;
-	font-family: courier;
-	margin: 0.5em;
-	display: flex;
-}
-
-table#header input {
-	font-size: 0.6em;
-	width: 6em;
-}
-
-button[type="submit"], .logout {
```

Comment:
<img width="613" alt="Screenshot 2024-01-25 at 23 07 20" src="https://github.com/underground-software/orbit/assets/1909841/932adbd5-78be-4413-a93b-c7e37c047e22">

<img width="1333" alt="Screenshot 2024-01-25 at 23 10 09" src="https://github.com/underground-software/orbit/assets/1909841/14184188-cdb9-4a0c-876f-99e05ec0bff5">


A bit cozy don't you think?